templates/rust/examples/foo_test.rs.twig

```diff
@@ -0,0 +1,74 @@
+//
+// WARNING! DO NOT RUN THIS TEST IF YOU ARE NOT USING THE APPWRITE SDK-GENERATOR TESTING STACK
+// IT WILL FAIL!
+//
+
+use std::path::PathBuf;
+use {{ language.params.packageName }};
+use reqwest::blocking::Response;
+use serde_derive::Deserialize;
+
+#[derive(Deserialize)]
+struct DefaultResponse {
+  result: Option<String>,
+  message: Option<String>
+}
+
+fn parse_response(data: Response) -> String {
+  let response:DefaultResponse = match data.json() {
+    Ok(value) => value,
+    Err(err) => {
+      return err.to_string()
+    }
+  };
+
+  match response.result {
+    Some(data) => return data,
+    None => return match response.message {
+      Some(value) => value,
+      None => "Couldn't parse error.".to_string()
+    }
+  }
+}
+
+fn main() {
+    let mut client = {{ language.params.packageName }}::client::Client::new();
+
+    client.set_endpoint("https://appwrite.io/");
+    client.set_project("");
+
+    let foo = {{ language.params.packageName }}::services::Foo::new(&client);
+    let bar = {{ language.params.packageName }}::services::Bar::new(&client);
+    let general = {{ language.params.packageName }}::services::General::new(&client);
+
```

Comment:
Also add a test-started message first in a new line which is what we use to check the actual test messages as some sdks output some extra outputs. Here's how we are doing for PHP
https://github.com/appwrite/sdk-generator/blob/feb6d752d9ac4d76c8b20995ff30b18b4870f778/tests/languages/php/test.php#L23
```suggestion

```